person.rb

```diff
@@ -19,3 +30,26 @@ def of_age?
     @age >= 18
   end
 end
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name
+  end
+end
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.capitalize
+  end
+end
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.strip
+  end
+end
```

Comment:
- [ ] Based on the requirement, the `correct_name` method of the `TrimmerDecorator` class should trim the name to the first ten characters if the name's length is more than `10` else, it should return the name as it is if the length is less than or equal to `10`. Kindly use this logic in the `correct_name` method for the `TrimmerDecorator` class. 😎 🖱️